lab-ron/index.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const greet = require(__dirname + '/lib/greet');
+
+//declare a const variable main
```

Comment:
If you're going with one line comments I'd add the one liner above the line it refers to, for example I'd put line 5 on line 2 in this case.  Great job though, your comments are helpful and terse.